build.sbt

```diff
@@ -28,16 +28,6 @@ scalacOptions ++= Seq(
   "-unchecked",
   "-Xlint",
   "-Xcheckinit",
-  "-Ywarn-adapted-args",
-  "-Ywarn-dead-code",
-  "-Ywarn-inaccessible",
-  "-Ywarn-dead-code",
-  "-Ywarn-nullary-override",
-  "-Ywarn-nullary-unit",
-  "-Ywarn-numeric-widen",
-  "-Ywarn-unused",
-  "-Ywarn-unused-import",
-  "-Ywarn-value-discard",
```

Comment:
Except the redundancy of some elemnts, why do you removed checks?

Reply:
they were deprecated in scala 2.12 and removed in scala 2.13

Reply:
Oh ok.